wingetui/uiSections.py

```diff
@@ -1919,7 +1931,30 @@ def __init__(self, parent = None):
         self.layout.addWidget(QLinkLabel(f"{_('Translators')}:", f"font-size: 22pt;font-family: \"{globals.dispfont}\";font-weight: bold;"))        
         self.layout.addWidget(QLinkLabel(_("WingetUI has not been machine translated. The following users have been in charge of the translations:")))
         translators = "<ul>"
-        for user in ("Aaron liu: Chinese", "Ahmet Özmetin: Turkish", "Artem Moldovanenko: Ukranian", "BUGP Association: Chinese", "Cololi: Chinese", "Datacra5H: German", "Evans: French", "GiacoBot: Italian", "gidano: Hungarian", "Maicol Battistini: Italian", "Nemanja Djurcic: Serbian, Croatian", "Operator404: Ukranian", "ppvnf: Portuguese (Portugal and Brazil)", "Rosario Di Mauro: Italian", "Satyam Singh Niranjan: Hindi", "Sergey: Russian", "sho9029: Japanese", "TAKASE, Yuki: Japanese", "yrctw: Chinese", "Артем: Russian", "Martí Climent: Catalan"):
+        for user in (
+            "Aaron liu: Chinese",
+            "Ahmet Özmetin: Turkish",
+            "Artem Moldovanenko: Ukranian",
+            "BUGP Association: Chinese",
+            "Cololi: Chinese",
+            "Datacra5H: German",
+            "Evans: French", "GiacoBot: Italian",
+            "gidano: Hungarian",
+            "Maicol Battistini: Italian",
+            "Martí Climent: Catalan",
+            "Nemanja Djurcic: Serbian, Croatian",
+            "Operator404: Ukranian",
+            "panther7: Czech",
+            "ppvnf: Portuguese (Portugal and Brazil)",
+            "regulargvy13: Polish",
+            "Rosario Di Mauro: Italian",
+            "Satyam Singh Niranjan: Hindi",
+            "Sergey: Russian",
+            "sho9029: Japanese",
+            "TAKASE, Yuki: Japanese",
+            "yrctw: Chinese",
+            "Артем: Russian",
+            ):
```

Comment:
TODO: Generate from lang files

Reply:
Via translation key, as "Translated to English by martinet101" in ElevenClock

Reply:
can you please open an issue regarding this?

Thanks!